google-cloud-datalabeling/pom.xml

```diff
@@ -58,6 +58,10 @@
       <groupId>com.google.api</groupId>
       <artifactId>gax-grpc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
```

Comment:
Please add `<scope>test</scope>` and move it with the other test dependencies

Reply:
Done